tool/merkle.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Please adjust this header: The description of the package shouldn't use `/* */` and instead just `//`.

Make a new line between `Package tool provides miscellneous functionality to deal with IOTA types.` and the above.

Check this out as a reference: https://github.com/golang/go/blob/master/src/compress/gzip/gunzip.go

Reply:
Why this was appoved then? https://github.com/iotaledger/iota.go/pull/163/files#diff-2399c9a33af208ffcfaf1b9c867265a3